### PR TITLE
Run semver check even when no prdoc

### PR DESCRIPTION
**Pull request by Morganamilo**

### CODE REVIEW

File: .github/workflows/check-semver.yml [hunk 1]
```diff
@@ -74,7 +72,7 @@ jobs:
 
       - name: install parity-publish
         # Set the target dir to cache the build.
-        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish -q
+        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish@0.8.0 -q
```
Comment (ggwpez) [inline .github/workflows/check-semver.yml:75]: 0.7.0 is still used somewhere else i think.

File: substrate/frame/src/lib.rs [hunk 2]
```diff
@@ -60,6 +60,7 @@
 //! }
 //! ```
 //!
+//! test
```
Comment (ggwpez) [inline substrate/frame/src/lib.rs:63]: Can you maybe try that the `bump: none` works now?

Comment (Morganamilo) [reply substrate/frame/src/lib.rs:63]: seems to work

File: .github/workflows/check-semver.yml (hunk 1 above)
Comment (Morganamilo) [reply .github/workflows/check-semver.yml:75]: Yeah will bump the others now that i confirmed it works

**Review by ggwpez — APPROVED** [review]

**Review by EgorPopelyaev — APPROVED** [review]

**Pull request merged**